docs/content/en/docs/user-guide/managing-piped/configuration-reference.md

```diff
@@ -49,6 +49,7 @@ spec:
 | hostName | string | The hostname or IP address of the remote git server. Default is the same value with Host. | No |
 | sshKeyFile | string | The path to the private ssh key file. This will be used to clone the source code of the specified git repositories. | No |
 | sshKeyData | string | Base64 encoded string of SSH key. | No |
+| enableNewBranch | bool | Whether to create a new branch or not when commit changes in event watcher. | No |
```

Comment:
Please edit a file in docs-dev.

Reply:
thanks! renamed and edited this in docs-dev.
[add createPullRequest flag to event watcher config instead of git config](https://github.com/pipe-cd/pipecd/pull/4395/commits/9f132b7eac0895e9e40dc9d230dd635b3888742d)